content/en/docs/kubernetes/operator/automatic.md

```diff
@@ -373,3 +375,175 @@ Alternatively, the annotation can be added to a namespace, which will result in
 for more details.
+
+## Troubleshooting
+
+If you run into problems trying to auto-instrument your code, here are a few
+things that you can try.
+
+### **1- Did the Instrumentation resource install?**
+
+After installing the `Instrumentation` resource, verify that it installed
+correctly by running this command, where `<namespace>` is the namespace in which
+the `Instrumentation` resource is deployed:
+
+```
+kubectl describe otelinst -n <namespace>
+```
+
+Sample output:
+
+```
+Name:         python-instrumentation
+Namespace:    application
+Labels:       app.kubernetes.io/managed-by=opentelemetry-operator
+Annotations:  instrumentation.opentelemetry.io/default-auto-instrumentation-apache-httpd-image:
+               ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-apache-httpd:1.0.3
+             instrumentation.opentelemetry.io/default-auto-instrumentation-dotnet-image:
+               ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-dotnet:0.7.0
+             instrumentation.opentelemetry.io/default-auto-instrumentation-go-image:
+               ghcr.io/open-telemetry/opentelemetry-go-instrumentation/autoinstrumentation-go:v0.2.1-alpha
+             instrumentation.opentelemetry.io/default-auto-instrumentation-java-image:
+               ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-java:1.26.0
+             instrumentation.opentelemetry.io/default-auto-instrumentation-nodejs-image:
+               ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-nodejs:0.40.0
+             instrumentation.opentelemetry.io/default-auto-instrumentation-python-image:
+               ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python:0.39b0
+API Version:  opentelemetry.io/v1alpha1
+Kind:         Instrumentation
+Metadata:
+ Creation Timestamp:  2023-07-28T03:42:12Z
+ Generation:          1
+ Resource Version:    3385
+ UID:                 646661d5-a8fc-4b64-80b7-8587c9865f53
+Spec:
+...
+ Exporter:
+   Endpoint:  http://otel-collector-collector.opentelemetry.svc.cluster.local:4318
+...
+ Propagators:
+   tracecontext
+   baggage
+ Python:
+   Image:  ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-python:0.39b0
+   Resource Requirements:
+     Limits:
+       Cpu:     500m
+       Memory:  32Mi
+     Requests:
+       Cpu:     50m
+       Memory:  32Mi
+ Resource:
+ Sampler:
+Events:  <none>
+```
+
+### **2- Do the OTel Operator logs show any auto-instrumentation errors?**
+
+Check the OTel Operator logs for any errors pertaining to auto-instrumentation
+by running this command:
+
+```
+kubectl logs -l app.kubernetes.io/name=opentelemetry-operator --container manager -n opentelemetry-operator-system --follow
+```
+
+### **3- Were the resources deployed in the right order?**
+
+Order matters! The `Instrumentation` resource needs to be deployed before before
+deploying the application, otherwise the auto-instrumentation won’t work.
```

Comment:
Is this true?  I thought if you had deployments with the annotation and created an Instrumentation object the operator would start its work.

Reply:
I tried it, and it does crap out if you deploy in the wrong order.